src/Cms/AppTranslations.php

```diff
@@ -88,6 +88,12 @@ protected function i18n(): void
 	 */
 	public function panelLanguage(): string
 	{
+		$translation = $this->request()->get('translation');
+
+		if ($translation && Translation::exists($translation)) {
```

Comment:
`Suggestion:` I like when we are really explicit what we check

```suggestion
		if ($translation !== null && Translation::exists($translation)) {
```